cassdegrees/templates/list.html

```diff
@@ -69,16 +69,21 @@
                 {# If there is no data available, notify the user #}
                 {% if not content %}
                     <p>No {{ title }}s to display</p>
+                    <div class="right">
+                        <p class="left marginright">
+                           <input class="btn-uni-grad btn-large" type ="submit" formaction="/{% if title == 'Degree' %}manage_programs{% elif title == 'Subplan' %}manage_subplans{% elif title == 'Course' %}manage_courses{% endif %}/?action=Add" value="Add a {{ title }}">
```

Comment:
I was thinking something along the lines of this.
1. user doesn't need to scroll down to add a course/sub-plan/program.
2. no duplicate code/ no need to worry about button alignment fo when there are/aren't any records to display.

I'm more than open to more suggestions.
![image](https://user-images.githubusercontent.com/24206502/56551556-0437a480-65cc-11e9-8a7d-864fc3402dac.png)

Reply:
This is pretty much what I did in my current uncommitted work. I'm happy to include this version since your reasoning is valid :)

Reply:
we could also consider floating action buttons so no matter where you are on the page, you have convenient access to the buttons